examples/discovery-search-app/cypress/integration/collection_filter/spec.ts

```diff
@@ -0,0 +1,142 @@
+          cy.get('.bx--list-box__menu-item')
+            .contains('finnegans wake')
+            .click();
+          cy.wait('@postQuerySingleCollection').as('singleCollectionQueryObject');
+        });
+
+        it('should make a query against only the selected collection', () => {
+          cy.get('@singleCollectionQueryObject')
+            //@ts-ignore
+            .its('requestBody.collection_ids')
+            .should('contain', 'paris19221939');
+        });
+
+        it('the clear all selected collections button appears', () => {
+          cy.get('div[aria-label="Clear Selection"]').should('exist');
+        });
+
+        describe('and we click the clear selected collections button', () => {
+          beforeEach(() => {
+            cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('postQuery');
+            cy.get('div[aria-label="Clear Selection"]').click();
+            cy.wait('@postQuery').as('clearedCollectionsQueryObject');
+          });
+
+          it('makes a query against all available collections', () => {
+            cy.get('@clearedCollectionsQueryObject')
+              //@ts-ignore
+              .its('requestBody.collection_ids')
+              .should('be.empty');
+          });
+        });
+
+        describe('and we select another collection', () => {
+          beforeEach(() => {
+            cy.route('POST', '**/query?version=2019-01-01', 'doubleCollectionQueryJSON').as(
+              'postQueryDoubleCollection'
+            );
+            cy.get('.bx--list-box__menu-item')
+              .contains('deadspin')
+              .click();
+            cy.wait('@postQueryDoubleCollection').as('doubleCollectionQueryObject');
+          });
+
+          it('makes a query against both of the selected collections', () => {
+            cy.get('@doubleCollectionQueryObject')
+              //@ts-ignore
+              .its('requestBody.collection_ids')
+              .should('contain', 'deadspin9876')
+              .and('contain', 'paris19221939');
+          });
+
+          describe('and we clear the selected collections', () => {
+            beforeEach(() => {
+              cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('postQuery');
+              cy.get('div[aria-label="Clear Selection"]').click();
+              cy.wait('@postQuery').as('originalQueryObject');
+            });
+
+            it('makes a query against all available collections', () => {
+              cy.get('@originalQueryObject')
+                //@ts-ignore
+                .its('requestBody.collection_ids')
+                .should('be.empty');
+            });
+          });
+        });
+      });
+
+      describe('and we click the selection box again', () => {
+        beforeEach(() => {
+          cy.get('#collection-facet-select').click();
+        });
+
+        it('the collection filter dropdown disappears', () => {
+          cy.get('#collection-facet-select__menu').should('not.exist');
```

Comment:
i would make sure something exists in tandem with the `not.exist` to make sure we aren't asserting a white screen is a pass here